newroutes/authenticated/sites.js

```diff
@@ -2,15 +2,11 @@ const autoBind = require("auto-bind")
 const express = require("express")
 
 // Import middleware
-const {
-  attachReadRouteHandlerWrapper,
-  attachWriteRouteHandlerWrapper,
-} = require("@middleware/routeHandler")
+const { attachReadRouteHandlerWrapper } = require("@middleware/routeHandler")
 
 class SitesRouter {
```

Comment:
Could I understand the reason for moving the handler into a new FormSG router? And by extension, the API endpoint of `POST /formsg/create-site`?

Reply:
I modified the top of this PR to explain why. Please let me know if you need more explanation.

Reply:
Copying my response from Slack to this PR:
Thanks for the explanation. I think that `POST /formsg/create-site` endpoint name isn't very intuitive because it's not clear what Formsg has to do with Isomer. 

I recommend keeping it to `POST /v2/sites` because the endpoint name is a better description of what the handler actually does; it creates a new Isomer site. I think it's fine for the `SiteRouter` to have methods with various middlewares.

Reply:
@alexanderleegs @seaerchin thoughts?

Reply:
I just discussed this with @prestonlimlianjie. I think he's about to leave another comment, but I'll attempt to summarize our discussion.

Having endpoints with different middleware in the same router would require some significant reworking of our routing architecture. It's hard to justify such reworking, since these FormSG endpoints may not be around for long. They were meant to be a stopgap measure until we can create sites in the CMS itself, hopefully in the next six months.

Reply:
Thanks for summarizing the discussion - the main reason for splitting out the new `createSite` handler into its own router is because the existing sites router uses the `authenticatedSubrouter` (`authenticatedSubrouter.use("/sites", sitesV2Router.getRouter())`). This `authenticatedSubrouter` requires all routes to use JWT authentication, which the current `createSite` handler does not require, or have access to, since the endpoint is called by FormSG.

We will refactor the `createSite` handler back into the sites router once we move the site creation flow into the CMS, where the handler will have access to the JWT.